hr_holidays_credit/models/hr_leave_type.py

```diff
@@ -64,10 +61,7 @@ def name_get(self):
                     )
 
             if extra:
-                record_name = _("%(name)s (%(extra)s)") % {
```

Comment:
Any specific reason changing this? Since that would affect existing translations and won't allow for RTL/LTR difference.

Reply:
OK, I changed back.